cmd/rpcdaemon/commands/engine_api.go

```diff
@@ -69,6 +69,16 @@ type TransitionConfiguration struct {
 	TerminalBlockNumber     *hexutil.Big   `json:"terminalBlockNumber"     gencodec:"required"`
 }
 
+type ExecutionPayloadBodyV1 struct {
+	Transactions [][]byte            `json:"transactions" gencodec:"required"`
+	Withdrawals  []*types.Withdrawal `json:"withdrawals"  gencodec:"required"`
+}
+
+type BeaconBlocksByRangeRequest struct {
+	StartSlot uint64 `json:"start_slot" gencodec:"required"`
```

Comment:
Slot numbers in CL are different from block numbers in the EL. This field refers to the block rather than the slot number (see Item 6 of [the spec](https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#specification-4)). Please rename to `StartBlockNumber` or simply `Start`. And it's "start" in json per the spec.

Reply:
https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#beaconblocksbyrange

This is where I got the name for the json from, it mentions not to confuse this with the block number which we're OK on.  I think we need to keep the `start_slot`, but can rename the field name no problem?

Reply:
The p2 spec is different fro the [Engine API spec](https://github.com/ethereum/execution-apis/blob/main/src/engine).

Reply:
Turns out we don't need this type at all anyway, the last comment here negates the need for it